### PR TITLE
export type PatternWhitelist in pkg/kubelet/sysctl

### PR DESCRIPTION
**Pull request by Aresforchina**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:
export type PatternWhitelist in pkg/kubelet/sysctl
**Which issue(s) this PR fixes**:

Ref #68026

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Review by mattjmcnaughton — APPROVED** [review]
/ok-to-test
/lgtm 

Marking as lgtm assuming all the tests pass.

**Comment by sftim** [comment]
Hi @Aresforchina - did you want me to review this? 
I'm not familiar with kubelet code I'm afraid.

**Comment by Aresforchina** [comment]
> Hi @Aresforchina - did you want me to review this?
> I'm not familiar with kubelet code I'm afraid.

No problem. Thank you all the same

File: pkg/kubelet/sysctl/whitelist.go [hunk 1]
```diff
@@ -30,19 +30,19 @@ const (
 	ForbiddenReason         = "SysctlForbidden"
 )
 
-// patternWhitelist takes a list of sysctls or sysctl patterns (ending in *) and
+// PatternWhitelist takes a list of sysctls or sysctl patterns (ending in *) and
```
Comment (liggitt) [inline pkg/kubelet/sysctl/whitelist.go]: exporting something just to satisfy golint doesn't make sense to me. this type still has private members which limits its usefulness as an exported type. I would probably switch to returning the interface from NewWhitelist instead if that would satisfy golint

**Pull request closed without merging**